src/convert.ts

```diff
@@ -2080,22 +2080,22 @@ export default function convert(config: ConvertConfig): ESTreeNode | null {
       break;
 
     case SyntaxKind.JsxSelfClosingElement: {
-      /**
-       * Convert SyntaxKind.JsxSelfClosingElement to SyntaxKind.JsxOpeningElement,
-       * TypeScript does not seem to have the idea of openingElement when tag is self-closing
-       */
-      (node as any).kind = SyntaxKind.JsxOpeningElement;
-
-      const openingElement = convertChild(node);
-      (openingElement as any).selfClosing = true;
-
       Object.assign(result, {
         type: AST_NODE_TYPES.JSXElement,
-        openingElement,
+        openingElement: {
```

Comment:
Please can we keep the original comment and just move it to here instead?

Reply:
ok